meet_balls_app/tests.py

```diff
@@ -195,3 +196,83 @@ def test_login_page_navigation(self, client):
         response = client.get(login_url)
         assert response.status_code == 200
         assert response.request['PATH_INFO'] == '/login/'
+
+
+@pytest.mark.django_db
+class TestProfile:
+
+    def test_profile_view_no_data(self, client, player):
```

Comment:
If I see it correctly, everything in this test, is also on the next test. Any added value in keeping this one?

Reply:
true, will be fixed